website/docs/r/kubernetes_cluster.html.markdown

```diff
@@ -136,6 +136,10 @@ resource "azurerm_subnet" "virtual" {
 
 * `windows_profile` - (Optional) A `windows_profile` block as defined below.
 
+* `private_link_enabled` (Optional) Enables private link for the AKS cluster, providing a private IP for the k8s api. Defaults to false.
```

Comment:
to match other resources could we make this:

```suggestion
* `private_link_enabled` (Optional) Should this Kubernetes Cluster have Private Link Enabled? This provides a Private IP Address for the Kubernetes API on the Virtual Network where the Kubernetes Cluster is located. Defaults to `false`. Changing this forces a new resource to be created.
```

Reply:
Resolved in new commit